SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LoadBalancing/WorkerRegion.h"
+
+#include "Materials/MaterialInstanceDynamic.h"
+#include "UObject/ConstructorHelpers.h"
+#include "UObject/UObjectGlobals.h"
+
+DEFINE_LOG_CATEGORY(LogWorkerRegion);
+
+const float AWorkerRegion::DEFAULT_WORKER_REGION_HEIGHT = 10;
+const float AWorkerRegion::DEFAULT_WORKER_REGION_OPACITY = 0.7;
+
+AWorkerRegion::AWorkerRegion(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	Mesh = ObjectInitializer.CreateDefaultSubobject<UStaticMeshComponent>(this, TEXT("WorkerRegionPlane"));
+	static ConstructorHelpers::FObjectFinder<UStaticMesh> PlaneAsset(TEXT("/Engine/BasicShapes/Plane.Plane"));
+	Mesh->SetStaticMesh(PlaneAsset.Object);
+	SetRootComponent(Mesh);
+}
+
+void AWorkerRegion::Init(UMaterial* Material, FColor Color, FBox2D Extents)
+{
+	this->SetHeight(AWorkerRegion::DEFAULT_WORKER_REGION_HEIGHT);
```

Comment:
Just interested why we need the "this->"?

Reply:
almost definitely won't 😅